src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import userService from '../../utils/user-service';
+import { ROUTES } from '../../shared/routes';
+
+const ProtectedRoute: React.FC<any> = ({ component: Component, ...rest }) => {
```

Comment:
Can't we fix this `any`?

Reply:
Fixed in: 490058b8c708b9b705a09044e59b350c11733afb 🍺